.github/workflows/push.yml

```diff
@@ -1,11 +1,9 @@
 name: CI
 on:
-  pull_request:
-    branches:
-      - main
+  push:
```

Comment:
Will this still trigger on pull requests after this change?

Reply:
Also it would seem all branches will now push to docker hub, is that the intention?

Reply:
Talked offline, and all commits that are pushed are to become images and for pull requests, we'll see if it works if not we'll do another change.